etc/base.yaml

```diff
@@ -178,17 +178,40 @@ postgresql:
         - name: POSTGRES_MULTIPLE_DATABASES
           value: restsourceauthorizer,appconfig,kratos # ... and these databases will be created
 
+cloudnative_postgresql:
+  _install: true
+  _chart_version: 0.1.0
+  _extra_timeout: 0
+  needs:
+    - cloudnative-pg
+#  Uncomment this when migrating from the legacy postgresql databases
```

Comment:
What would happen if these steps are enabled by default?

Reply:
It would try to ingest data from a non existing upstream source. Most likely, erroring out the `initdb` pod that is started. Would you argue to have this always on?